templates/react/packages/client/mud/contractComponents.ts

```diff
@@ -0,0 +1,24 @@
+/* Autogenerated file. Do not edit manually. */
+
+import { TableId } from "@latticexyz/utils";
+import { defineComponent, Type as RecsType, World } from "@latticexyz/recs";
+
+export function defineContractComponents(world: World) {
```

Comment:
weird that this was missing before and not caught by CI

Reply:
ahh, because it wasn't supposed to be here: https://github.com/latticexyz/mud/pull/877